src/components/Collection/CleanDescription.tsx

```diff
@@ -2,21 +2,23 @@ import React, { useMemo } from 'react';
 import cx from 'classnames';
 
 // The question marks are there because people can't spell…
-const RemoveSummaryRegex = /\b(Sour?ce|Note|Summ?ary):([^\r\n]+|$)/mg;
+const CleanInfoLinesRegex = /\b((Modified )?Sour?ce|Note( [1-9])?|Summ?ary):(?!$)([^\r\n]+|$)/img;
 
-const RemoveBasedOnWrittenByRegex = /^(\*|\u2014) (based on|written by) ([^\r\n]+|$)/img;
-
-const RemoveBBCodeRegex = /\[i\](.*?)\[\/i\]/sg;
+// eslint-disable-next-line operator-linebreak -- Because dprint and eslint can't agree otherwise. Feel free to fix it.
+const CleanMiscLinesRegex =
+  /^(\*|\u2014 (adapted|source:?|summary|translated|written)|- (translated)|~ (adapted|description|summary|translated)) ([^\r\n]+|$)/img;
```

Comment:
Which character is unicode 2014, and why is it needed here? 

Also just combine the unicode, dash, and tilde into a single 1 character block, and the other words into another, and allow the colon to be present for al- _you know what, I'll just give you a code suggestion instead of trying to explain it in plain English. Here;
```suggestion
  /^(\*|[\u2014\-~] (adapted|source|description|summary|translated|written):?) ([^\r\n]+|$)/img;
```

Reply:
u2014 is an emdash (—) which accounts for ~300 descriptions which have written by entries (my example file above only listed the first match for each regexp per description).

The suggested changes do make it slightly less clear which keywords apply to which special character but I do prefer it like this thanks.